base/ltexpl.dtx

```diff
@@ -549,4 +549,26 @@
 %<latexrelease>\EndIncludeInRelease
 %    \end{macrocode}
 %
+% \begin{macro}{\expandableinput}
+% \changes{v1.3i}{2025/02/26}{Added document level name for \cs{file_input_raw:n} (gh/514)}
```

Comment:
if it is called raw here then perhaps \rawinput might be a better choice on the document level.

Reply:
Was my initial thinking, but I wanted to see how it looked to others - and we had said 'expandable input' as a no (capitalisation not specified!)

Reply:
"expandable" seems to make more sense than "rawl, does it not?

Reply:
Oops: should be "raw" !

Reply:
As we discussed in the team meeting, getting general users to understand when they might need something expandable is hard - so I don't think there is a perfect name. I can live with `\expandableinput`, `\ExpandableInput` or `\rawinput` (or some others - things like `\untrackedinput` or `\lowlevelinput` would also make some sense). Basically we have to pick _something_ :)

Reply:
I think I'd prefer `\ExpandableInput` (camel case like `\InputIfFileExists`)  `\UntrackedInput` would also work.

Reply:
I tend to agree with David: but using such a name name needs a very clear explanation of what it does, and does not, do! Including a reference to the name!

Reply:
@car222222 I added something I hope is clear to `usrguide`